docs/reference/access-log-policy.md

```diff
@@ -0,0 +1,106 @@
+# Access Log Policy API Reference
+
+## Introduction
+
+The Access Log Policy (ALP) custom resource allows you to define access logging configurations on
```

Comment:
I actually prefer using full AccessLogPolicy (without whitespace) overall because this abbreviation is not an official thing in VPC Lattice. what do you think?

Reply:
Fair enough, will switch next commit